benchmarks/asv.conf.json

```diff
@@ -6,6 +6,10 @@
     "environment_type": "conda-delegated",
     "show_commit_url": "https://github.com/scitools/iris/commit/",
     "branches": ["upstream/main"],
+    "build_command": [
+        "python setup.py build",
+        "python -mpip wheel --no-deps -w {build_cache_dir} {build_dir}"
+    ],
```

Comment:
ASV has now adopted default commands that will build in the absence of `setup.py` - using only `pyproject.toml` for instructions. This relies on the **build** package being in the environment.

Since Iris is still using `setup.py` and doesn't have **build** as a dependency: I have restored the previous behaviour.

Changes in `setup.py` may make the commands here fail, so I have set benchmarks to trigger if `setup.py` has any changes.